xstate/state.py

```diff
@@ -10,16 +8,16 @@
 
 
 class State:
-    configuration: Set[StateNode]
+    configuration: Set["StateNode"]
```

Comment:
Curious: why strings?

Reply:
It's either that or `from __future__ import annotations` but the latter is 3.7+

https://www.python.org/dev/peps/pep-0484/#forward-references

https://www.python.org/dev/peps/pep-0563/

Reply:
Luckily autocomplete still work